docs/dev/libvirt-howto.md

```diff
@@ -286,6 +286,32 @@ kubectl get --all-namespaces pods
 ## Troubleshooting
 If following the above steps hasn't quite worked, please review this section for well known issues.
 
+### console doesn't come up
+In case of libvirt there is no wildcard dns resolution and console is depend on the route which is created by auth operator.
+To make it work we need to use the multi-stage installation and edit the `domain` for ingress.
+
+Following steps are required to have console operator up and working.
+```console
+// Add another domain entry in the openshift.conf which used by dnsmasq
+// Here `tt.testing` is the domain which I choose when running the installer.
+$ cat /etc/NetworkManager/dnsmasq.d/openshift.conf
+server=/tt.testing/192.168.126.1
+address=/.apps.tt.testing/192.168.126.51
+
+// Make sure you restart the NetworkManager after change in the openshift.conf
+$ sudo systemctl restart NetworkManager
+
+// Create the manifests
+$ openshift-install create manifests --dir myTestDir
+
+// Edit the cluster-ingress-02-config.yml file
```

Comment:
but the following command is not editing the file so comment is a bit confusing.

Reply:
@zeenix Edited.